components/dashboard/src/components/UsageView.tsx

```diff
@@ -36,22 +40,37 @@ function UsageView({ attributionId }: UsageViewProps) {
     const today = new Date();
     const startOfCurrentMonth = new Date(today.getFullYear(), today.getMonth(), 1);
```

Comment:
[drive-by] Could be simplified to `dayjs().startOf('month')`, we already do this [here](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/components/UsageBasedBillingConfig.tsx#L48)